src/serac/physics/solid_mechanics.hpp

```diff
@@ -720,7 +720,8 @@ class SolidMechanics<order, dim, Parameters<parameter_space...>, std::integer_se
                                                              // fact that the displacement, acceleration, and shape
                                                              // fields are always-on and come first, so the `n`th
                                                              // parameter will actually be argument `n + NUM_STATE_VARS`
-        [this, material](auto /*x*/, auto& state, auto displacement, auto acceleration, auto shape, auto... params) {
+        [this, material](double /*t*/, auto /*x*/, auto& state, auto displacement, auto acceleration, auto shape,
```

Comment:
I think not passing time into this lambda is OK as we should not allow time-dependent material properties.

Reply:
what about viscoelastic or other rate-dependent models?

Reply:
Rate is OK, but absolute time doesn't make sense. For example, we would want to have a bulk modulus that is a function of time.

Reply:
Capturing an offline discussion here - passing time to calculate the timestep size (and therefore the rate quantities) is a possible path forward for rate-dependent materials. However, adding that capability is well outside the scope of this PR.

Reply:
FWIW, I agree with Jamie's points.

Reply:
One could envision aging models that have an explicit dependence on time.  But certainly not a use case we have at the moment.